Maven Build Failure Due to Missing AspectJ Weaver JAR Dependency

During the build of the treblle-data-publisher project, the following error was encountered while running: `mvn clean install`

```
[ERROR] dependency: org.aspectj:aspectjweaver:jar:1.8.5 (system)
[ERROR] Could not find artifact org.aspectj:aspectjweaver:jar:1.8.5 at specified path /Users/rahhulk7/Downloads/lib/aspectjweaver-1.8.5.jar
```
## Steps to Reproduce:

Clone the repository or set up the project.
Run the Maven build using the command:

`mvn install`

The build fails with the error related to the aspectjweaver JAR file at a hard-coded system path.

### Expected Result:
The build should complete successfully without issues related to missing dependencies.

### Actual Result:
The build fails due to the missing aspectjweaver dependency at the specified system path.

## Solution:

The error occurs because the pom.xml specifies the aspectjweaver JAR dependency with a hard-coded path, which is unsuitable for cross-platform development. The issue can be resolved by:

1. Deleting the existing dependency from the local Maven repository:

`rm -rf ~/.m2/repository/org/aspectj/aspectjweaver/1.8.5`

2. Updating the pom.xml to use a proper version of the aspectjweaver dependency, such as 1.9.9, and changing the scope from system to compile:

```
<dependency>
    <groupId>org.aspectj</groupId>
    <artifactId>aspectjweaver</artifactId>
    <version>1.9.9</version> <!-- New version -->
    <scope>compile</scope>
</dependency>
```

Rebuild the project:

`mvn clean install`

This should resolve the issue and allow the Maven build to proceed successfully.

### Environment:
Maven Version: 3.9.9
OS: macOS (Apple Silicon)
WSO2 API Manager Version: 4.3.0
Treblle WSO2 SDK: treblle-data-publisher:bundle:1.0.0

--- a/wso2am-4.3.0/pom.xml
+++ b/wso2am-4.3.0/pom.xml
@@ -1,3 +1,5 @@
+Pom.xml
+
 <?xml version="1.0" encoding="UTF-8"?>
 
 <project xmlns="http://maven.apache.org/POM/4.0.0"
@@ -26,6 +28,12 @@
             <artifactId>synapse-core</artifactId>
             <version>${wso2.synapse.version}</version>
             <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.aspectj</groupId>
+            <artifactId>aspectjweaver</artifactId>
+            <version>1.9.19</version>
         </dependency>
 
         <dependency>
@@ -129,7 +137,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>2.0</version>
+                <version>3.11.0</version>
                 <configuration>
                     <source>1.8</source>
                     <target>1.8</target>

